docs/ubuntu1604.md

```diff
@@ -0,0 +1,22 @@
+Ubuntu 16.04 prep
+-----------------
+
+The following commands are one possible way to quickly prepare an instance of
+Ubuntu 16.04 for Uno.
+
+```bash
+#install stuff
+sudo apt install --assume-yes openjdk-8-jdk maven git openssh-server wget
+
+#setup passwordless ssh
+ssh-keygen -t rsa -N '' -f ~/.ssh/id_rsa
```

Comment:
This might override user's own key. Should warn or use a custom location. It's actually not necessary to set up passwordless ssh, if user uses ssh-agent to preload key with password.